lib/arjdbc/common_jdbc_methods.rb

```diff
@@ -1,27 +0,0 @@
-module ArJdbc
-  # This is minimum amount of code neede from base JDBC Adapter class to make common adapters
-  # work.  This replaces using jdbc/adapter as a base class for all adapters.
-  module CommonJdbcMethods
-    def initialize(connection, logger = nil, config = {})
-      config[:adapter_spec] = adapter_spec(config) unless config.key?(:adapter_spec)
-
-      connection ||= jdbc_connection_class(config[:adapter_spec]).new(config, self)
-
-      super(connection, logger, config)
-    end
-
-    def translate_exception(e, message)
-      # we shall not translate native "Java" exceptions as they might
-      # swallow an ArJdbc / driver bug into a AR::StatementInvalid ...
-      return e if e.is_a?(NativeException) # JRuby 1.6
```

Comment:
I don't think this NativeException line is needed if it is we should change the comment since JRuby 1.6 will never be able to run Rails 5.x

Reply:
I just copied this over from the jdbc adapter so I'm good with removing it if it isn't needed, I wasn't sure what it was catching so I didn't want to touch it :)

Reply:
... can pbly go for 5.x (needing JRuby 9K anyway) but the constant is still there so it won't fail :)

Reply:
@kares yeah true I am just trying to remove potentially unknown-reason code from this branch.  If it is due to our pre-NativeException case in older JRuby we should remove it and just see what breaks.